dali/base/dadfs.cpp

```diff
@@ -4884,7 +4884,48 @@ protected: friend class CDistributedFilePart;
             calcFileCost(cluster, sizeGB, fileAgeDays, numDiskWrites, numDiskReads, atRestCost, accessCost);
         }
     }
-};
+    // getUnmountedPath - returns false if can't return unmounted path
+    virtual bool getUnmountedPath(unsigned partNum, StringBuffer & path) override
+    {
+        IClusterInfo &cluster = clusters.item(0);
+        const char *planeName = cluster.queryGroupName();
+        if (isEmptyString(planeName))
+            return false;
+        Owned<IStoragePlane> storagePlane = getDataStoragePlane(planeName, false);
+        if (!storagePlane)
+            return false;
+
+        StringBuffer planePrefix(storagePlane->queryPrefix());
+        Owned<IStoragePlaneAlias> alias = storagePlane->getAliasMatch(accessMode);
+        if (alias)
+        {
+            StringBuffer newPlanePrefix(alias->queryPrefix());
+            if (startsWith(path, newPlanePrefix))
+                planePrefix.swapWith(newPlanePrefix);
+        }
+        assertex(startsWith(path, planePrefix));
+        unsigned numStripedDevices = storagePlane->numDevices();
+        unsigned mountPathLength = planePrefix.length();
+        assertex(isPathSepChar(path.charAt(mountPathLength)));
+        ++mountPathLength;
+        if (numStripedDevices > 1)
+        {
+            assertex(path.charAt(mountPathLength)=='d');
+            ++mountPathLength;
+            unsigned stripeNum = calcStripeNumber(partNum, lfnHash, numStripedDevices);
```

Comment:
Simpler for this code to just skip digits - and not worry about how many there were.